src/vivarium/framework/values.py

```diff
@@ -111,10 +111,15 @@ def rescale_post_processor(value: NumberLike, manager: "ValuesManager") -> Numbe
 
     """
     if hasattr(value, "index"):
-        time_step = manager.simulant_step_sizes(value.index).dt
+        return value.mul(
+            manager.simulant_step_sizes(value.index)
+            .astype("timedelta64[ns]")
+            .dt.total_seconds()
+            / (60 * 60 * 24 * 365.0),
```

Comment:
We're inconsistent with 365 vs 365.25 days per year - is this appropriate?

Reply:
I agree that this is problematic, though this is an inconsistency that I have simply preserved from `from_yearly`. I think it probably makes sense to change both that and this to 365.25